### PR TITLE
[140] Add UITest template and update install script

### PR DESCRIPTION
**Pull request by blyscuit**
https://github.com/nimblehq/ios-templates/issues/140

## What happened

Add Templates for UITest Protocols and Screen Specs.

Add installation prompt to `install.sh`

## Insight

Some files need to be moved manually.

Some files need to be added to Main Target manually.

## Proof Of Work

https://user-images.githubusercontent.com/6356137/117247510-9257ee00-ae68-11eb-886b-95d7603578d7.mp4

https://user-images.githubusercontent.com/6356137/117247515-9421b180-ae68-11eb-9b74-8d2b651f96f7.mp4

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -56,6 +56,26 @@ if [[ $response = 1 ]]; then
   success "Copied Nimble Templates successfully"
 fi
 
+# ----------------------------------
+# UITest Templates
+# ----------------------------------
+section "UITest Templates"
+confirm "Do you want to copy UITest Templates?"
```
Comment (vnntsu) [inline install.sh:63]: Please make a suggestion for developers (e.g `1/0` or `yes/no`)

**Review by vnntsu — APPROVED** [review]
the rest looks good

File: UITest Templates/UITestProtocols.xctemplate/AccessibilityIdentifiable.swift [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        dictionary.forEach(setAccessibilityIdentifier)
+    }
+}
+
+extension AccessibilityIdentifiable where Self: UIViewController {
+
+    func setAccessibilityIdentifierDomainToView() {
+        view.accessibilityIdentifier = Identifier.screenName
+    }
+}
```
Comment (suho) [inline UITest Templates/UITestProtocols.xctemplate/AccessibilityIdentifiable.swift:36]: I think we should put these codes into the project as default, don't need to put them into `xctemplate`

Comment (suho) [reply UITest Templates/UITestProtocols.xctemplate/AccessibilityIdentifiable.swift:36]: We only need to create files for the test module: `___VARIABLE_moduleName___FlowSpec.swift`, `___VARIABLE_moduleName___Identifier.swift`,  `___VARIABLE_moduleName___Screen.swift `

**Review by suho — COMMENTED** [review]
please check my comments

File: UITest Templates/UITestProtocols.xctemplate/AccessibilityIdentifiable.swift (hunk 2 above)
Comment (blyscuit) [reply UITest Templates/UITestProtocols.xctemplate/AccessibilityIdentifiable.swift:36]: My inten

File: install.sh (hunk 1 above)
Comment (blyscuit) [reply install.sh:63]: This is taken care by Mark's helper 
Screen Shot 2021-07-06 at 15 59 46

File: UITest Templates/UITestProtocols.xctemplate/AccessibilityIdentifiable.swift (hunk 2 above)
Comment (suho) [reply UITest Templates/UITestProtocols.xctemplate/AccessibilityIdentifiable.swift:36]: @blyscuit I think we can create a framework, or put these files as default into the project

**Review by suho — APPROVED** [review]

**Review by nmint8m — APPROVED** [review]
LGTM

**Pull request merged**